app/models/article.rb

```diff
@@ -35,7 +35,9 @@ def name
   end
 
   def path
-    if published?
+    if published? && short_path
```

Comment:
`short_path` shouldn't be in play here. 

`path` is the *canonical` path/url for an Article/Page/etc.

`short_path` should only be used to create Redirects.

***

Sorry that there aren't better specs / definition around this feature (and others). I've mostly been working alone and have too much tribal knowledge in my brain and not enough documentation. Classic. 

If you stick around for a while, you'll get a better lay of the land. And hopefully we can get better specs and docs written.